### ISSUE TITLE
Collection count appears to be off

### ISSUE THREAD

**Issue by pconwell** [kowsheek/nft-gamestop-marketplace-metrics#10]
As of this post, your [webapp ](https://sevenfourone.live/)shows 477 collections. However, gamestop's [website ](https://nft.gamestop.com/explore) shows 274.

**Comment by kowsheek** [comment]
The GameStop website shouls 274/5 collections per page. I'm getting all collections.

**Comment by pconwell** [comment]
Not sure why you closed it so quickly. I'm counting 5 full pages of 48 collections each, plus one partial page of 34 collections. (5 * 48) + 34 = 274 collection total.

**Comment by kowsheek** [comment]
Sorry, clicked the wrong button 😅

I am seeing unique collections IDs from the API, that's what the number is based off of.

**Comment by pconwell** [comment]
There was one collection added since I posted this, so it's now 275.

https://api.nft.gamestop.com/nft-svc-marketplace/getCollectionsPaginated?&limit=0 shows `{"data":[],"totalNum":275,"limit":0,"offset":0}`. You can confirm this by setting the limit above 275, say 9999, and count the collection IDs  inside the {"data"} tag: https://api.nft.gamestop.com/nft-svc-marketplace/getCollectionsPaginated?&limit=9999 .

(side note, it appears the API is hardcoded to a limit of `limit=500` if you put a number larger than 500 in. Once collections get above 500, you'll have to step through with `offset=...`.)

**Comment by kowsheek** [comment]
I hear you, I'm not sure what the difference between that API and the one I'm using to get all collections `https://api.nft.gamestop.com/nft-svc-marketplace/getCollections` is.

**Comment by pconwell** [comment]
Hmm... quite the mystery. This API (https://api.nft.gamestop.com/nft-svc-marketplace/getCollectionsPaginated?&limit=0) and the website both show 275, but this API (https://api.nft.gamestop.com/nft-svc-marketplace/getCollections) shows 480...

Let me see if I can compare the two lists to see what collectionIDs appear in both lists versus what's missing from one or the other.

**Comment by pconwell** [comment]
I cannot 100% confirm this, but I went through my list of 275 collectionIds and they *all* had at least 1 NFT listed in their collection. I then went through a sample of collectionIds that were in your list but not in mine (the collections I'm "missing"). All of them I checked had zero NFTs listed in the collection - they were empty. Random example: https://nft.gamestop.com/collection/punchingpandas

So, I can't say for sure, but I'm pretty confident that the API endpoint I'm using only includes collections with content, while the other API endpoint contains *all* collections, including empty collections.
